src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (clone.hasOwnProperty(newKey)) {
```

Comment:
Instead of using 'hasOwnProperty', which is an older approach, you can use the 'in' operator or 'Object.prototype.hasOwnProperty.call' for checking property existence to avoid issues with properties that may shadow object's own properties.